ipaserver/install/krbinstance.py

```diff
@@ -451,6 +451,30 @@ def pkinit_enable(self):
         service.set_service_entry_config(
             'KDC', self.fqdn, [PKINIT_ENABLED], self.suffix)
 
+    def pkinit_disable(self):
+        """
+        unadvertise enabled PKINIT feature in master's KDC entry in LDAP
+        """
+        ldap = api.Backend.ldap2
+        dn = DN(('cn', 'KDC'),
+                ('cn', self.fqdn),
```

Comment:
This is how the DN is constructed throughout installers (including `set_service_entry_config` which you yourself added recently), so I'm not changing it.